src/test/java/microsoft/exchange/webservices/data/property/complex/ComplexPropertyCollectionTest.java

```diff
@@ -0,0 +1,53 @@
+package microsoft.exchange.webservices.data.property.complex;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.List;
+
+/**
+ * @author Vladislav Bauer
```

Comment:
please don't add author tag since often classes will be handled by several contributors